src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/EffectiveFlowDirectionExtensions.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
```

Comment:
EffectiveFlowDirection is a Maui.Controls concept - it's not part of Maui (which will only have FlowDirection). None of the EffectiveFlowDirection stuff should be at this level. 

It looks like it's only needed for `AlignmentExtensions.ToNative()` on iOS - just change it to a bool `isLtr` and pass in `false` until we introduce FlowDirection at the IFrameworkElement level (we have a [PR](https://github.com/xamarin/Xamarin.Forms/pull/13394), it's just not reviewed yet).

Reply:
Fixed it up while rebasing.